docs/source/extensions.rst

```diff
@@ -69,13 +69,8 @@ connection via .env file, environment variables or settings.py::
         },
     }
 
-The user role must have at least the privileges
-
-* SELECT
-* INSERT
-* UPDATE
-* DELETE
-* CONNECT
+The user role must have at least the privileges SELECT, INSERT, UPDATE, DELETE,
+CONNECT
```

Comment:
It is an enumeration of privileges. The previous format looked like small headings when rendered in the documentation to me - I didn't like it.

Reply:
And exactly because it is an enumeration a list would make the most sense actually.
It is weird that it looked like small headings. But then I understand your will of change.